.github/workflows/ci.yml

```diff
@@ -20,3 +20,16 @@ jobs:
           coverage: xdebug
       - run: composer install
       - run: vendor/bin/phpunit --coverage-text
+
+  static-analysis:
+    name: PHPStan
+    runs-on: ubuntu-20.04
+    continue-on-error: true
+    steps:
+      - uses: actions/checkout@v3
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.1
+      - run: composer require phpstan/phpstan
+      - name: Execute type checking
+        run: vendor/bin/phpstan --configuration="phpstan.types.neon.dist"
```

Comment:
Should we move this file to the `tests/` folder or add the file to the `.gitattributes`?

Reply:
Should we rename this to `phpstan.neon.dist` to make PHPStan pick the default path instead of explicitly giving this argument? https://phpstan.org/config-reference#config-file

Reply:
No, yes, and yes.